src/discord-cluster-manager/consts.py

```diff
@@ -52,3 +52,4 @@ class ModalGPU(Enum):
 POSTGRES_USER = os.getenv("POSTGRES_USER")
 POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD")
 POSTGRES_PORT = os.getenv("POSTGRES_PORT")
```

Comment:
We probably won't need the `POSTGRES_*` constants, but I wanted to leave them for now in order to verify we can connect to the database on Heroku. Once we verify that, we should be able to remove these constants.